src/Misc/MiddleWare.cpp

```diff
@@ -506,7 +525,7 @@ class mw_dispatcher_t : public master_dispatcher_t
     mw_dispatcher_t(MiddleWare* mw) : mw(mw) {}
 };
 
-class MiddleWareImpl
+class MiddleWareImpl : public WaveTableRequestHandler
```

Comment:
It seems like a MiddleWareImpl has a wavetable request handler relationship is more fitting than middlewareimpl is a wavetable request handler.